frontend/src/metabase/admin/TroubleshootingApp.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { getTroubleshootingInfo } from 'metabase/admin/admin'
+
+import LoadingAndErrorWrapper from 'metabase/components/LoadingAndErrorWrapper'
+
+const mapStateToProps = state => ({
+    troubleShootingInfo: state.admin.troubleshooting
+})
+
+const mapDispatchToProps = {
+    getTroubleshootingInfo
+}
+
+class TroubleshootingApp extends Component {
+    componentWillMount () {
+        console.log('Requested state')
+        this.props.getTroubleshootingInfo()
+    }
+    render () {
+        return (
+            <LoadingAndErrorWrapper loading={!this.props.troubleShootingInfo}>
+                { this.props.troubleShootingInfo && 
```

Comment:
The fact that this isn't taken care of by the LoadingAndErrorWrapper makes me peeved. If you don't explicitly guard against nulls, you get null pointer errors.